<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>虚拟体温计</string>
  </property>
  <property name="windowIcon">
   <iconset theme="temperature-warm"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>-10</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#001900;&quot;&gt;虚拟体温计&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>599</width>
       <height>129</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="checkBox_use_current_time">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>301</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>使用当前时间作为随机数种子</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>114514</number>
      </property>
      <property name="value">
       <number>114514</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>114514</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发烧</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>599</width>
       <height>129</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>测量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>体温：</string>
      </property>
     </widget>
     <widget class="QLabel" name="temperature_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string> </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>警告：此程序仅供娱乐，生成的是随机数，请配合防疫工作，上报真实体温

此程序是自由软件；您可以以自由软件基金会发布的 GNU
通用公共许可协议第三版或(您可以选择)更高版方式重新发
布它和/或修改它。
此程序是希望其会有用而发布，但没有任何担保；没有
甚至是暗含的适宜销售或特定目的适用性方面的担保。详情参看GNU 通用公共许可协议。
您应该与此程序一道收到了一份 GUN 通用公共许可协议的
副本；如果没有，请访问 &lt;https://www.gnu.org/licenses/gpl-3.0.html&gt;</string>
    </property>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>label</zorder>
   <zorder>scrollArea_2</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
